### ISSUE TITLE
[$20][MSFT-138] [Register] Some social media hyperlinks missed by the screen reader

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2834]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member/registration
2. Switch on the screen reader
3. Press Tab Key and Listen

**Expected Result**
It should be read-out by the screen reader.

**Actual Result**
Some social media hyperlinks missed by the screen reader 
 - GITHUB
 - GOOGLE
 - FACEBOOK

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=6aa53cde%2D1eef%2D4bae%2Dbfe5%2Df5f5f79e9df4&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F48

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096266 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096266 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096266 has been updated - it has been assigned to Timor.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by romitgit** [comment]
PR for this issue : https://github.com/appirio-tech/accounts-app/pull/244

**Comment by drasticdpk** [comment]
Operating System: Windows
Browser: Chrome
Tested URL : https://accounts.topcoder-dev.com/member/registration
Screen Reader : JAWS

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097198 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097198 has been updated - it has been assigned to Timor.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by drasticdpk** [comment]
Passed:
Windows: 
Browser : Chrome/Edge/Firefox

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097198<br/><br/>```This is an automated message for crazyk via Topcoder X```
